### PR TITLE
truncate: add page

### PR DESCRIPTION
**Pull request by felixonmars**
- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/linux/truncate.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+- Set the size of an existing file, or create a new file with the specified size:
+
+`truncate -s 10G {{filename}}`
+
+- Extend the file size, fill with holes (which reads as zero bytes):
+
+`truncate -s +50M {{filename}}`
+
+- Shrink the file and discard the extra data:
```
Comment (agnivade) [inline pages/linux/truncate.md]: I am guessing it removes data from the end ? We should clarify that.

File: pages/linux/truncate.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# truncate
+
+> Shrink or extend the size of a file to the specified size.
+
+- Set the size of an existing file, or create a new file with the specified size:
```
Comment (agnivade) [inline pages/linux/truncate.md]: We should mention the size here. `Set a size of 10 GB to an exsting file, or .. `

File: pages/linux/truncate.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+`truncate -s 10G {{filename}}`
+
+- Extend the file size, fill with holes (which reads as zero bytes):
```
Comment (agnivade) [inline pages/linux/truncate.md]: Same as previous description.

File: pages/linux/truncate.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/truncate.md]: Hrm. How to word this?  Maybe `Shrink the file by 2 GiB, starting from the end of the file. Will delete data when shrinking:`?

Comment (agnivade) [reply pages/linux/truncate.md]: `Shrink the file by 2GiB, by removing data from the end of file:` ?

**Comment by felixonmars** [comment]
Sorry for the slow response, I'll update soon.

File: pages/linux/truncate.md (hunk 1 above)
Comment (sbrl) [reply pages/linux/truncate.md]: Yeah!

**Comment by felixonmars** [comment]
Updated accordingly. Forgot again :S

**Comment by felixonmars** [comment]
Looks like my push forces another pull request instead of revive this one: https://github.com/tldr-pages/tldr/pull/2196

**Pull request closed without merging**